File: src/main/java/org/app/app1/java/Point.java
package org.app.app1.java;

public class Point {
    private int x;
    private int y;

    public Point(int x, int y) {
        this.x = x;
        this.y = y;
    }

    @Override
    public String toString() {
        return "(" + x + ", " + y + ")";
    }

    public void move(int dx, int dy) {
        this.x += dx;
        this.y += dy;
    }
}

File: src/main/java/org/app/app2/scala/Point.scala
package org.app.app2.scala

/**
  * Created by sarrafan on 5/9/17.
  */
class Point(var x: Int, var y: Int) {
  def move(dx: Int, dy: Int): Unit = {
    x = x + dx
    y = y + dy
  }

  override def toString: String =
    s"($x, $y)"
}